pkg/authenticator/config/config.go

```diff
@@ -143,5 +149,17 @@ func populateConfig() (*Config, error) {
 		defaultConfig.ClientCertPath = envVal
 	}
 
+	// Parse client cert retry count limit if one is provided from env
```

Comment:
should we have a test for this in config_test?

Reply:
i looked at this a bit. we have this logic also in another instance but a bit different. i don't want to waste much time on this so i'll leave it out of scope.

Reply:
this concerns me - the config.go is poorly unit tested, and it's not going to get better if we add new logic and don't add unit tests for it.